src/targets/browser/index.jsx

```diff
@@ -77,7 +77,7 @@ const renderApp = function(appLocale, client, isPublic) {
                     <SharingProvider
                       doctype="io.cozy.files"
                       documentType="Notes"
-                      previewPath="/preview/"
+                      previewPath="/preview"
```

Comment:
Let's use the `SHARING_LOCATION` var instead?

Reply:
Yes for sure